src/main/kotlin/no/nb/bikube/core/model/TitleDto.kt

```diff
@@ -0,0 +1,12 @@
+package no.nb.bikube.core.model
+
+import kotlinx.serialization.SerialName
+import kotlinx.serialization.Serializable
+
+@Serializable
+class TitleDto(
```

Comment:
Must add some info at some point. Publisher, publishplace, language, medium=text and dates are already "in use" in Collection.

Reply:
Will add